### PR TITLE
feat: add spacing + button + input + textarea tokens

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Tokens for:

- Spacing
- Button
- Input
- Textarea

### CODE REVIEW

File: tokens/spacing/base.json [hunk 1]
```diff
@@ -1,16 +1,32 @@
 {
-  "space": ""
+  "spacing": {
+    "space-1/2": {
```
Comment (delisma) [inline tokens/spacing/base.json:3]: We should be using a naming convention from 100-900 and then use the space-1/2 in the components tokens structure instead.

**Review by delisma — CHANGES_REQUESTED** [review]

File: tokens/components/button/base.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
+{
+  "button": {
+    "danger": {
+      "default": {
+        "value": "$colors.red.200",
+        "type": "button"
```
Comment (delisma) [inline tokens/components/button/base.json:6]: Type is still attached to the category not the component. In this case it's still linked to the color category.

File: tokens/components/button/base.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          }
+        },
+        "active": {
+          "background": {
+            "value": "$colors.blue.900",
+            "type": "button"
+          }
+        }
+      },
+      "outline": {
```
Comment (delisma) [inline tokens/components/button/base.json:36]: We should aligned with the design decisions on buttons:
The solid variant would be the only variant included in the MVP for sake of simplicity and avoiding breaking change with the data model in the future (no outline or text only style at the moment)

File: tokens/components/input/base.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+  "input": {
+    "danger": {
+      "default": {
+        "value": "$colors.red.300",
+        "type": "input"
```
Comment (delisma) [inline tokens/components/input/base.json:6]: Type is still attached to the category not the component. In this case it's still linked to the color category.

File: tokens/components/textarea/base.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
+  "textarea": {
+    "danger": {
+      "default": {
+        "value": "$colors.red.300",
+        "type": "textarea"
```
Comment (delisma) [inline tokens/components/textarea/base.json:6]: Type is still attached to the category not the component. In this case it's still linked to the color category.

**Review by delisma — CHANGES_REQUESTED** [review]

File: tokens/components/button/base.json (hunk 3 above)
Comment (melaniebmn) [reply tokens/components/button/base.json]: @delisma I removed all outline and text-only button tokens.

File: tokens/components/input/base.json (hunk 4 above)
Comment (melaniebmn) [reply tokens/components/input/base.json]: @delisma Type is updated to color.

File: tokens/components/textarea/base.json (hunk 5 above)
Comment (melaniebmn) [reply tokens/components/textarea/base.json]: @delisma Type is updated to color

File: tokens/components/button/base.json (hunk 2 above)
Comment (melaniebmn) [reply tokens/components/button/base.json]: @delisma Type is updated to color.

**Comment by melaniebmn** [comment]
@delisma can you review the PR again pls? I updated it with your requested changes.

**Pull request closed without merging**